src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -82,6 +89,10 @@ operator()(const NodeID, const EdgeID source_edge_id, Intersection intersection)
                 intersection_node_id, intersection[1], intersection_node_one);
             const auto intersection_following_index_two = findNextIntersectionForRoad(
                 intersection_node_id, intersection[2], intersection_node_two);
+            // in case of broken roads, we return
+            if (intersection_following_index_one.empty() ||
+                intersection_following_index_two.empty())
```

Comment:
Thats the result of the above addition. Theoretically, we should never be able to reach this if not for other problems. But right now our demo-server is seeing endless loops and the early return from the in my view only possible location results in empty state here.